src/js/stories/PlaceholderStories.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import {
+    PlaceholderTable,
+    PlaceholderTextCell as TextCell,
+    PlaceholderIconCell as IconCell,
+    PlaceholderText,
+    PlaceholderIcon,
+} from '../components';
+import { JTable, TableRow, TableCell } from '../components/';
+
+
+storiesOf('Placeholder', module)
+    .add('PlaceholderTable', () => <TableStory />)
+    .add('using JTable', () => <JTableStory />)
+;
+
+const style = {
+    padding: 10,
+};
+
+
+function TableStory() {
+    const columns = [
+        { width: 75, head: { text: 50 }, cell: { icon: 20 } },
+        { width: 250, isFlexible: true, head: { text: 50 }, cell: { text: 200 } },
+        { width: 125, head: { text: 50 }, cell: { text: 100 } },
+        { width: 125, head: { text: 50 }, cell: { text: 100 } },
+        { width: 50, head: { text: 50 }, cell: { icon: 20 } },
+    ];
+
+    return (
+        <div style={style}>
+            <p>PlaceholderTable driven by 'columns' prop</p>
+
+            <PlaceholderTable columns={columns} style={{maxHeight: 415}} />
```

Comment:
Not sure what the `maxHeight` is for? Is it a leftover from dev / testing? I kind of like the idea of having the placeholder cut off, but a fade would be a lot nicer that a chop mid-row.